src/cephadm/cephadm

```diff
@@ -4189,6 +4190,11 @@ def _get_parser():
         type=int,
         default=DEFAULT_RETRY,
         help='max number of retries')
+    parser.add_argument(
```

Comment:
intentionally making this arg global for all commands?  with this PR only `shell` and `ceph-volume` commands would make use of the arg ...

Reply:
yes, _right now_ it's only `shell` and `ceph-volume` that use the `--env` args, but I can foresee that we'll need this for other subcommands in the future aswell.
The mgr/cephadm implementation also allows to pass arbitrary environment variables down to bin/cephadm now.

Reply:
yeah that's a good point. Might be interesting to add this during `cephadm deploy` and than rework this commit: 93533e8a5fc1dcc4c98fadfd2661cf3fd6f68bf8